code/modules/mob/living/silicon/robot/drone/drone_items.dm

```diff
@@ -8,7 +8,6 @@
 
 	//Has a list of items that it can hold.
 	var/list/can_hold = list(
-		/obj/item/stock_parts/cell,
```

Comment:
Why was this removed?

Reply:
`/obj/item/stock_parts`, is also listed, so it's already there.

Reply:
Oh yeah this is fine then.